website/docs/r/server.html.markdown

```diff
@@ -34,6 +34,7 @@ The following arguments are supported:
 * `image` - (Required) base image of server
 * `type` - (Required) type of server
 * `bootscript` - (Optional) server bootscript
+* `boot_type` - (Optional) server boot_type (`local`, `bootscript`)
```

Comment:
might be worth making this `the boot mechanism for this server. Possible values include `bootscript` and `local`